rollup/transaction_ingestion.go

```diff
@@ -0,0 +1,181 @@
+package rollup
+
```

Comment:
The various hardfork heights will change behavior of the codebase, we need to make sure that all of the hardforks are turned on from the beginning so that its just like ethereum execution today. The gas pricing of opcodes could be off at low heights.

Reply:
Running with clique will activate all of the hardforks from genesis, so we don't have to worry about this.